pytest.ini

```diff
@@ -1,3 +1,5 @@
 [pytest]
-DJANGO_SETTINGS_MODULE = thr.settings
-python_files = tests.py test_*.py *_tests.py
\ No newline at end of file
+DJANGO_SETTINGS_MODULE = thr.settings.test
+python_files = tests.py test_*.py *_tests.py
+filterwarnings =
+    ignore::DeprecationWarning
```

Comment:
🤔

Reply:
This was used to hide this warning:
```
========================================= warnings summary =========================================
trackhubs/api/tests.py::test_post_trackhub_success
  /Users/username/git/thr/thr_env/lib/python3.8/site-packages/django_elasticsearch_dsl/fields.py:126: 
DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' 
is deprecated since Python 3.3, and in 3.9 it will stop working
    if isinstance(objs, collections.Iterable):

-- Docs: https://docs.pytest.org/en/stable/warnings.html
```
This warning is shown because the `django-elasticsearch-dsl` package used (compatible with ES version `>=6.0.0 and <7.0.0`) is a bit behind.
I commented that line to keep the warning.